src/configure/clients/azure/armRestClient.ts

```diff
@@ -10,6 +10,17 @@ export class ArmRestClient {
         this.restClient = new RestClient(azureSession.credentials);
     }
 
+    public fetchArmData(endPointUri: string, httpMethod: string, body?: string){
```

Comment:
should have a space before the ending `{`. Have you turned off auto-formatting in vs code?

Reply:
fixed it.